rust/src/clib/lib.rs

```diff
@@ -26,6 +26,8 @@ const NMSTATE_FLAG_RUNNING_CONFIG_ONLY: u32 = 1 << 7;
 const NMSTATE_PASS: c_int = 0;
 const NMSTATE_FAIL: c_int = 1;
 
+const VERSION: Option<&str> = option_env!("CARGO_PKG_VERSION");
```

Comment:
Please just use `const VERSION: &str = "2.1.0"`;

Reply:
Is not better to try it from cargo pkg ?